router/core/errors.go

```diff
@@ -197,17 +197,15 @@ func writeRequestErrors(r *http.Request, w http.ResponseWriter, statusCode int,
 // writeOperationError writes the given error to the http.ResponseWriter but evaluates the error type first.
 // It also logs additional information about the error.
 func writeOperationError(r *http.Request, w http.ResponseWriter, requestLogger *zap.Logger, err error) {
+	requestLogger.Debug("operation error", zap.Error(err))
+
 	var reportErr ReportError
 	var httpErr HttpError
 	var poNotFoundErr *persistedoperation.PersistentOperationNotFoundError
 	switch {
 	case errors.As(err, &httpErr):
-		requestLogger.Debug(httpErr.Error())
 		writeRequestErrors(r, w, httpErr.StatusCode(), graphqlerrors.RequestErrorsFromError(err), requestLogger)
 	case errors.As(err, &poNotFoundErr):
-		requestLogger.Debug("persisted operation not found",
```

Comment:
This information is also already logged in the `PersistentOperationNotFoundError` error message.